File: Assets/AltUnityTester-BindingsAndExamples(can_be_deleted)/Bindings~/python/altunityrunner/commands/ObjectCommands/get_text.py
from altunityrunner.commands.base_command import BaseCommand
class GetText(BaseCommand):
    def __init__(self, socket,request_separator,request_end,alt_object):
        super().__init__(socket,request_separator,request_end)
        self.alt_object=alt_object

    def execute(self):
        data = self.send_data(self.create_command('getText', self.alt_object))
        return self.handle_errors(data)

File: Assets/AltUnityTester/AltUnityDriver/Commands/ObjectCommands/GetText.cs
public class GetText : AltBaseCommand
{
  AltUnityObject altUnityObject;
  
  public GetText(SocketSettings socketSettings, AltUnityObject altUnityObject) : base(socketSettings)
  {
    this.altUnityObject = altUnityObject;
  }
  
  public string Execute()
  {
    string altObject = Newtonsoft.Json.JsonConvert.SerializeObject(altUnityObject);
    Socket.Client.Send(System.Text.Encoding.ASCII.GetBytes(CreateCommand("getText", altObject)));
    string data = Recvall();
    if (!data.Contains("error:")) return data;
    HandleErrors(data);
    return null;
  }
}